drake/systems/plants/collision/ResultCollector.cpp

```diff
@@ -0,0 +1,19 @@
+#include <iostream>
+
+#include "drake/systems/plants/collision/DrakeCollision.h"
+#include "ResultCollector.h"
```

Comment:
:grin:

I'll fix some of these (in both the new `ResultCollector.{cpp,h}` and in `BulletResultCollector.{cpp,h}` where they originated. And by "some" I think I mean all of them except the naming convention. Stay tuned...